### ISSUE TITLE
[Adaptive Cards - ExpenseReport]: Talkback is announcing irrelevant information while activating the expand button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#100]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'ExpenseReport'.
8. Navigate to 'Air Travel expense' button and double tap to activate it.
9. Observe the issue.

**Actual Result:**
While navigating to 'Air Travel Expense 300' button after activating it, Talkback is announcing irrelevant information as **'Card Content 1 expended chevron up 1 expanded chevron down collapsed'.**

**Expected Result:**
After activaitng the 'Air Travel Expense' button, Talkback should not convey the irrelevant information and when user expand it then it should announce as **Air Travel Expenses 300 button expanded** and when user collapsed it then it should announce as **'Air Travel Expenses 300 button collapsed'**.

**User Impact:**
It will impact the screen reader user as they will get confused and they will not able to know whether same button is expanded or not.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships
Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/66f443a0-2769-4de8-a505-02e60b95b33f

![Talkback is announcing irrelevant information while activating the expand button](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/7fc97309-a331-40a8-9a83-51f8c4b76849)

### Card JSON

```JSON
NA
```
